src/i18n/en.js

```diff
@@ -14,6 +14,8 @@ export default {
       "lock.network": "We could not reach the server. Please check your connection and try again.",
       "lock.popup_closed": "Popup window closed. Try again.",
       "lock.unauthorized": "Permissions were not granted. Try again.",
+      "lock.mfa_registration_required": "Multifactor authentication is required but your device is not enrolled. Please enroll it before moving on.",
```

Comment:
Not sure what to say in this case; we don't support enrollment. "Enrollment not supported"?